cloverage/test/cloverage/coverage_test.clj

```diff
@@ -81,9 +81,8 @@
                    wrapped))))
 
 (t/deftest test-wrap-list
-  (t/is (expand= `(cov/capture 0 ((cov/capture 1 +) (cov/capture 2 1)
-                                                    (cov/capture 3 2)))
-                 (inst/wrap #'cov/track-coverage 0 `(+ 1 2)))))
+  (t/is (expand= `(cov/capture 0 ((cov/capture 1 my-fn) (cov/capture 2 1) (cov/capture 3 2)))
+                 (inst/wrap #'cov/track-coverage 0 `(my-fn 1 2)))))
```

Comment:
Tweaked because the instrumented version of `(+ 1 2)` is no longer `((capture ... +) (capture ... 1) (capture ... 2))`